utils/ssh/ssh.go

```diff
@@ -80,40 +80,47 @@ type Client struct {
 }
 
 func NewSSHClientWithCluster(cluster *v1.Cluster) (*Client, error) {
-	var host string
-	if cluster.Spec.Provider == common.CONTAINER {
-		host = cluster.Spec.Masters.IPList[0]
-	}
+	var (
+		ipList []string
+		host   string
+	)
+	sshClient := NewSSHByCluster(cluster)
 	if cluster.Spec.Provider == common.AliCloud {
 		host = cluster.GetAnnotationsByKey(common.Eip)
+		if host == "" {
+			return nil, fmt.Errorf("get cluster EIP failed")
+		}
+		ipList = append(ipList, host)
+	} else {
+		ipList = append(ipList, append(cluster.Spec.Masters.IPList, cluster.Spec.Nodes.IPList...)...)
 	}
-	sshClient := NewSSHByCluster(cluster)
-	if sshClient == nil {
-		return nil, fmt.Errorf("cloud build init ssh client failed")
-	}
-
-	if host == "" {
-		return nil, fmt.Errorf("get cluster EIP failed")
+	err := WaitSSHReady(sshClient, 6, ipList...)
+	if err != nil {
+		return nil, err
 	}
-	err := WaitSSHReady(sshClient, host)
+	uLimit := `if [ "$(ulimit -n)" -lt "65535" ];then echo "* - nofile 65535" >> /etc/security/limits.conf; fi`
```

Comment:
Add ulimit in NewSSHClient is very not friendly, The best way to config ulimit is using plugin.

Reply:
> Add ulimit in NewSSHClient is very not friendly, The best way to config ulimit is using plugin.

well